src/py42/clients/detectionlists/departing_employee.py

```diff
@@ -15,15 +15,15 @@ def __init__(self, session, user_context, detection_list_user_client):
         self._detection_list_user_client = detection_list_user_client
 
     def add(self, user_id, departure_date=None):
-        """Adds a user to the Departing Employees list. Creates a detection list user profile if one
-         didn't already exist.
+        """Adds a user to the Departing Employees list. Creates a detection list user profile if one \
```

Comment:
I made it so that documentation build warnings are now errors, which prompted these changes.